src/main/java/uk/gov/companieshouse/pscstatementdataapi/services/PscStatementService.java

```diff
@@ -113,7 +132,7 @@ private void saveToDb(String contextId, String companyNumber, String statementId
 
   private Created getCreatedFromCurrentRecord(String companyNumber,String statementId) {
     Optional<PscStatementDocument> document = pscStatementRepository.getPscStatementByCompanyNumberAndStatementId(companyNumber, statementId);
-    return document.isPresent() ? document.get().getCreated(): null;
+    return document.map(PscStatementDocument::getCreated).orElse(null);
```

Comment:
Happy for this to stay in but could we add a note for testing to run the PUT automation since it will affect that?

Reply:
I'll revert this change as it is not relevant to this Jira - don't want to create more testing work